### PR TITLE
Fix typo in GNOME mediakeys service name

### PR DESCRIPTION
**Pull request by JasonLG1979**

### CODE REVIEW

File: pithos/plugins/mediakeys.py [hunk 1]
```diff
@@ -28,8 +28,9 @@ class MediaKeyPlugin(PithosPlugin):
     method = None
     mediakeys = None
     de_busnames = [
-        ('gnome', 'org.gnome.SettingDaemon.MediaKeys'),
+        ('gnome', 'org.gnome.SettingsDaemon.MediaKeys'),
         ('gnome', 'org.gnome.SettingsDaemon'),
+        ('mate', 'org.mate.SettingsDaemon.MediaKeys'),
```
Comment (TingPing) [inline pithos/plugins/mediakeys.py]: Do you know if they actually changed this?

**Comment by JasonLG1979** [comment]
I'll fire up a few distros this weekend in VMs to see what the heck is going on. I'd like to get this figured out for the up coming bug fix release.

**Comment by JasonLG1979** [comment]
I'm currently downloading the Xfce, MATE, Cinnamon and LXDE Fedora 27 spins. I think that should provide pretty good coverage.

**Comment by TingPing** [comment]
@JasonLG1979 Anything else you want in a bugfix release?

**Comment by JasonLG1979** [comment]
I'd like to see if I can't fix mediakeys. So far I've tested MATE and not much has changed. It still doesn't work the same as GNOME. Connecting to 'notify::is-active' works though. I'd like to test the other spins and see how they behave.

**Comment by JasonLG1979** [comment]
Give me a couple hours and if I can't figure something out go ahead and do the release.

**Comment by JasonLG1979** [comment]
My testing methodology is:
1. Install and update the spin in GNOME boxes.
2. Enable/install rpmfusion repos.
3. Install d-feet to poke around DBus.
4. Install git and Pithos deps as listed in the wiki.
5. Clone, build and install git master of Pithos as instructed in the wiki.
6. See how the DE deals with mediakeys by opening both Pithos and another media player that has mediakeys support.

**Pull request merged**